pkgs/by-name/tr/translate/package.nix

```
@@ -0,0 +1,33 @@
+{
+  darwin,
+  fetchFromGitHub,
+  lib,
+  libiconv,
+  rustPlatform,
+  stdenv,
+}:
+rustPlatform.buildRustPackage {
+  pname = "tl";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "NewDawn0";
+    repo = "tl";
+    rev = "v1.0.0";
+    sha256 = "sha256-6oKz68UIRLRB7l5Am9+GNHswJKIrV/31dhkLRPC6dhE=";
+  };
+  cargoHash = "sha256-QEEqkcsJulZtMpVZXqy5D187nA+ksRya8ggPB9YWILU=";
+  buildInputs = [
+    libiconv
+  ] ++ lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.SystemConfiguration;
```

It's no longer necessary to specify Apple SDK components

```suggestion
  ];
```